### PR TITLE
jmxweb plugin version 0.0.1

### PR DESCRIPTION
**Pull request by deleolajide**
New plugin that implements a web console for openfire jmx using hawt.io and jolokia
See https://community.igniterealtime.org/blogs/ignite/2015/01/10/openfire-jmx-web-console

### CODE REVIEW

File: src/plugins/jmxweb/changelog.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            font-size : 100%;
+        }
+    </style>
+</head>
+<body>
+
+<h1>
+</h1>
+
+<p><b>0.0.0.1 -- January 9th 2015</p>
```
Comment (Redor) [inline src/plugins/jmxweb/changelog.html]: Should that not be 0.0.1?

**Comment by deleolajide** [comment]
Well spotted :-) thanks. yes indeed it should be 0.0.1

**Comment by Redor** [comment]
Looks fine to me.

**Comment by tevans** [comment]
Looks good to me also. Thanks Dele!

**Comment by deleolajide** [comment]
Looks like I have to merge this myself. If there are no objections, I will  do it over the weekend

**Comment by tevans** [comment]
Not seeing any conflicts, so I will merge it for you. Thanks for your excellent work!

**Comment by akrherz** [comment]
Dele, bamboo failed to [build](http://bamboo.igniterealtime.org/download/OPENFIRE-NIGHTLYPLUGINS-JOB1/build_logs/OPENFIRE-NIGHTLYPLUGINS-JOB1-1494.log) this

```text
build	29-Jan-2015 12:40:59	     [copy] Copying 19 files to /home/j2ee-bamboo/bamboo-agent-home/xml-data/build-dir/OPENFIRE-NIGHTLYPLUGINS-JOB1/work/plugins-dev/jmxweb/target/web
build	29-Jan-2015 12:40:59	 [trycatch] Caught exception: java.lang.NoClassDefFoundError: org/apache/xpath/XPathAPI
build	29-Jan-2015 12:40:59	     [echo] Error building plugin: jmxweb. Exception:
build	29-Jan-2015 12:40:59	     [echo] /home/j2ee-bamboo/bamboo-agent-home/xml-data/build-dir/OPENFIRE-NIGHTLYPLUGINS-JOB1/build/build.xml:1496: java.lang.NoClassDefFoundError: org/apache/xpath/XPathAPI
```

**Comment by deleolajide** [comment]
I have seen this problem before in an openfire build. Sometime it wants xalan.jar in the build/lib/ant folder and sometimes it doesn't. Travis builds ok without it.

I have xalan-2.7.1.jar on mine just in case. Should I commit it to openfire git?

**Comment by akrherz** [comment]
@deleolajide thanks for the response. travis-CI actually failed, but was not reported due to how we were invoking ant.  sure, adding another PR and then testing travis-CI with it would be great

**Comment by deleolajide** [comment]
@daryl, done on same PR. It was still open 
@tom, thanks for merging

**Pull request merged**